ouroboros-consensus/bench/PerasCertDB-bench/Main.hs

```diff
@@ -0,0 +1,102 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE LambdaCase #-}
+
+-- | This module contains benchmarks for Peras chain weight calculation as
+--   implemented by the by the
+--   'Ouroboros.Consensus.Peras.Weight.weightBoostOfFragment' function.
+--
+--   We benchmark the calculation on a static sequence of chain fragments of increasing
```

Comment:
Wouldn't it make sense to refer to `benchmarkParams` directly. Otherwise we'll have to modify this comment as well if we alter said variable.